defaults/main.yml

```diff
@@ -20,8 +20,8 @@ repox_path_log: /var/log
 repox_path_tomcat: /usr/share/tomcat
 
 # the user that logs in to the web interface
```

Comment:
Where are admin_username and admin_password coming from? If your goal is to not have a default (which is perfectly reasonable), then let's not set the vars in this file at all.